internal/cmd/flink/command_shell.go

```diff
@@ -21,7 +21,8 @@ func (c *command) newShellCommand(prerunner pcmd.PreRunner) *cobra.Command {
 			return c.startFlinkSqlClient(prerunner, cmd)
 		},
 	}
-	cmd.Flags().String("compute-pool", "", "Flink compute pool ID.")
+
+	c.addComputePoolFlag(cmd)
 	cmd.Flags().String("identity-pool", "", "Identity pool ID.")
```

Comment:
are `compute-pool` and `identity-pool` at similar level? Could make the format consistent (use a `addFlag` too)

Reply:
Identity pool has no autocompletion, and is only used in this function, so we don't need a helper function here yet.